pkg/cloud/services/iam/reconcilers.go

```diff
@@ -0,0 +1,180 @@
+package iam
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	v1certmanager "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+	"sigs.k8s.io/cluster-api-provider-aws/v2/cmd/clusterawsadm/converters"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+// reconcilePodIdentityWebhook generates certs and starts the webhook in the workload cluster
+// https://github.com/aws/amazon-eks-pod-identity-webhook
+// 1. generate webhook certs via cert-manager in the management cluster
```

Comment:
@luthermonson 
What is the reasoning behind creating certs resources in the management cluster?

Reply:
Security. You don't want unencrypted traffic to an HTTP webhook endpoint.

Reply:
ehn, i think i know what he's asking... @pavansokkenagaraj capa does not automatically install cert-manager into the downstream cluster. for capi projects the management cluster gets cert-manager automatically and is the only place we can generate them which is why it's done there and moved